workloads/files/workload-http-script-cm.yml

```diff
@@ -0,0 +1,1217 @@
+        die 1 "Cannot create wlg-targets ConfigMap for WLG pod(s)."
+      oc create secret generic wlg-ssh-key --from-file=wlg-ssh-key=$SERVER_RESULTS_SSH_KEY -n=$centos_stress_ns || \
+        die 1 "Cannot create wlg-ssh-key Secret for WLG pod(s)."
+      oc create cm k8s-config --from-file=k8s-config=$k8s_config -n=$centos_stress_ns || \
+        die 1 "Cannot create k8s-config ConfigMap for WLG pod(s)."
+
+      test "$LOAD_GENERATOR_NODES" && check_admin && {
+        # Lode generator nodes were specified and we have a cluster admin, steer the workload generator
+        NODE_SELECTOR='{"test": "wlg"}'
+      }
+
+      # create all the workload generators
+      while test $wlg -le ${LOAD_GENERATORS:-1}
+      do
+        oc process \
+          -pIDENTIFIER=$wlg \
+          -pNODE_SELECTOR="${NODE_SELECTOR:-null}" \
+          -f $wlg_config \
+          -n=$centos_stress_ns | \
+            oc create -f- -n=$centos_stress_ns &
+        sleep 0.1s
+        wlg=$(($wlg + 1))
+      done
+    }
+
+    oc_cleanup() {
+      exec 3>&1 4>&2 >/dev/null 2>&1
+      oc delete pods -n=$centos_stress_ns --all
+      oc delete cm -n=$centos_stress_ns --all      # wlg-targets! (e.g. switch from http->edge)
+      oc delete secret wlg-ssh-key -n=$centos_stress_ns || true
+      oc delete ns $centos_stress_ns || true
+      exec 2>&4 1>&3
+    }
+
+    client_wait_complete() {
+      local completed
+      while true; do
+        completed=$(oc get pods --no-headers -n=$centos_stress_ns -l=app=centos-stress --field-selector=status.phase=Succeeded 2>/dev/null | wc -l)
+        echo "Completed wlg pods: $completed/${LOAD_GENERATORS}"
+        test $completed -eq ${LOAD_GENERATORS} && break
+        sleep 10
+      done
+    }
+
+    oc_cleanup
+    oc_create
+    client_wait_complete
+    oc_cleanup
+
+  ###################################################################################################
+  stress-pod.yaml: |
+    apiVersion: v1
+    kind: Template
+    metadata:
+      name: centos-stress-template
+    objects:
+    - apiVersion: v1
+      kind: Pod
+      metadata:
+        generateName: centos-stress-
+        labels:
+          app: centos-stress
+      spec:
+        affinity:
+          podAntiAffinity:
+            requiredDuringSchedulingIgnoredDuringExecution:
+            - labelSelector:
+                matchExpressions:
+                - key: app
+                  operator: In
+                  values:
+                  - centos-stress
+              topologyKey: kubernetes.io/hostname
+        containers:
+        - env:
+          - name: IDENTIFIER
+            value: ${IDENTIFIER}
+          - name: NAMESPACE
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.namespace
+          envFrom:
+          - configMapRef:
+              name: wlg-env
+          image: quay.io/jmencak/centos-stress
```

Comment:
I think we should consider making this image configurable in the event we want to more quickly use a different stress container image.

Reply:
Agreed.